src/plugins/intel_npu/src/plugin/src/metrics.cpp

```diff
@@ -39,6 +44,13 @@ Metrics::Metrics(const std::shared_ptr<const NPUBackends>& backends) : _backends
                             ov::hint::num_requests.name(),
                             ov::intel_npu::compilation_mode_params.name(),
                             ov::intel_npu::dynamic_shape_to_static.name()};
+
+    // optimizationCapabilities: append dynamic features based on compiler version
+    uint32_t compilerversion = GetCompilerVersion();
+    // Dynamic quantization supported starting from compiler version x
+    if (compilerversion >= ONEAPI_MAKE_VERSION(6, 3)) {
```

Comment:
Let's put here a version which will for sure have MLIR DQ that can be enabled for all

Reply:
It shouldn't be X+1 but some super version where we'll claim that the below feature is generally supported.
What means - once the feature is generally supported, the driver version MUST be reported to the magic number we put here (e.g., `9, 9`).